pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -2346,7 +2369,7 @@ func TestRunBindPlugins(t *testing.T) {
 				SchedulerName: testProfileName,
 				Plugins:       plugins,
 			}
-			fwk, err := newFrameworkWithQueueSortAndBind(r, profile, withMetricsRecorder(recorder))
+			fwk, err := newFrameworkWithQueueSortAndBind(r, profile, stopCh, withMetricsRecorder(recorder))
```

Comment:
better to move `defer close(stopCh)` just after `stopCh := make(chan struct{})`. 
`stopCh` won't be closed when `if err != nil {}`